projects/Mallard/src/components/article/types/article.tsx

```diff
@@ -284,8 +284,17 @@ const Article = ({
 	};
 
 	const handleLightbox = (parsed: LightboxMessage) => {
-		const index = parsed.index;
+		let index = parsed.index;
 
+		if (
```

Comment:
It would be good to add a comment so in the future we know why we have this complex `if` statement.

Reply:
Good idea! will do that now